### ISSUE TITLE
Why are method2 and method 2 mutually exclusive?

### ISSUE THREAD

**Issue by vsaase** [JuliaIO/NIfTI.jl#26]
https://github.com/JuliaIO/NIfTI.jl/blob/a9e50b9f6ddf5e8b7c09fffdfe9a3d48a962c0fe/src/NIfTI.jl#L344

Hi, the nifti standard (https://nifti.nimh.nih.gov/nifti-1/documentation/nifti1fields/nifti1fields_pages/qsform.html) does'nt say that.

**Comment by Tokazama** [comment]
You're correct. However, last I checked ITK only uses the sform. They may have changed this though. Either way we need to fix this.
